NewSpring/ApollosProject
# improvement: Expands Short Links

## DESCRIPTION

![Screen Shot 2019-11-13 at 9 20 03 AM](https://user-images.githubusercontent.com/2659478/68772142-4b809380-05f7-11ea-9e39-8707c7302a52.png)


### What does this PR do, or why is it needed?

Will try to expand slug as a short link first to get the content

### How do I test this PR?

Click different links to make sure they open in the app. On alpha, a short link you can use is 'emmaguthrie'.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
